src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2731,6 +2731,17 @@ components:
         description:
           type: string
           description: Free text that describes the profile
+        cloud_platform:
+          $ref: '#/components/schemas/CloudPlatform'
+        tenant:
+          type: string
+          description: An optional tenant id for Azure
```

Comment:
Could you use the `UniqueIdProperty` property for these fields instead of checking if they are valid UUIDs in the validator?